vg/tee.go

```diff
@@ -0,0 +1,139 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package vg
+
+import (
+	"image"
+	"image/color"
+)
+
+// MultiCanvas creates a canvas that duplicates its drawing operations to all
+// the provided canvases, similar to the Unix tee(1) command.
+//
+// Each drawing operation is sent to each listed canvas, one at a time.
+func MultiCanvas(cs ...Canvas) Canvas {
+	return teeCanvas{cs}
+}
+
+type teeCanvas struct {
+	cs []Canvas
+}
+
+// SetLineWidth sets the width of stroked paths.
+// If the width is not positive then stroked lines
+// are not drawn.
+//
+// The initial line width is 1 point.
+func (tee teeCanvas) SetLineWidth(w Length) {
+	for _, c := range tee.cs {
+		c.SetLineWidth(w)
+	}
+}
+
+// SetLineDash sets the dash pattern for lines.
+// The pattern slice specifies the lengths of
+// alternating dashes and gaps, and the offset
+// specifies the distance into the dash pattern
+// to start the dash.
+//
+// The initial dash pattern is a solid line.
+func (tee teeCanvas) SetLineDash(pattern []Length, offset Length) {
+	for _, c := range tee.cs {
+		c.SetLineDash(pattern, offset)
+	}
+}
+
+// SetColor sets the current drawing color.
+// Note that fill color and stroke color are
+// the same, so if you want different fill
+// and stroke colors then you must set a color,
+// draw fills, set a new color and then draw lines.
+//
+// The initial color is black.  If SetColor is
+// called with a nil color then black is used.
+func (tee teeCanvas) SetColor(c color.Color) {
+	for _, canvas := range tee.cs {
+		canvas.SetColor(c)
+	}
+}
+
+// Rotate applies a rotation transform to the
+// context.  The parameter is specified in
```

Comment:
Reduce double space to single.

Reply:
(also removed from the original `vg.Canvas` documentation)